change/@ni-nimble-components-5084859c-b0ef-4431-b684-b4733c6d7714.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Seems reasonable. @atmgrifter00 can you review as select / combobox expert and see if other edge cases, etc, in the logic to verify?